src/components/probeHybridisation/ProbeAddPanel.tsx

```diff
@@ -145,7 +186,15 @@ const ProbeAddPanel = ({ probePanels }: ProbeLotAddPanelProps) => {
                 values.labware.forEach((lw, index) => {
                   const updatedLabware: ProbeOperationLabware = {
                     ...lw,
-                    probes: [...lw.probes, { name: probeName, lot: probeLot, plex: Number(probePlex) }]
+                    probes: [
+                      ...lw.probes,
+                      {
+                        name: probeName,
+                        lot: probeLot,
+                        plex: Number(probePlex),
+                        costing: probeCosting === 'SGP' ? SlideCosting.Sgp : SlideCosting.Faculty
```

Comment:
As probeCosting can be empty as well, it is better to compare with correct value other than ternary operation.

Reply:
probeCosting is a required field can not be empty, validation will not pass until the user selects a costing

Reply:
Ok